h/search/query.py

```diff
@@ -89,6 +107,20 @@ def __call__(self, search, params):
                  "unmapped_type": "boolean"}}
         )
 
+    def _date_parser(self, str_value):
```

Comment:
Since this is a function rather than a class, I would suggest the name should read as an action rather than a noun - ie. `_parse_date`.

It would also be useful to add a docstring clarifying what is returned (ms since epoch) and more importantly _why_ (this is how ES stores dates internally, and that is what _search_after_ values are matched against).